** (describe-copying) uses /usr/share/common-licenses/GPL
   On Debian systems, the full text of the GPL is available from the
   /usr/share/common-licenses directory.  The COPYING file included
   in the Emacs distribution is not installed in `data-directory',
   so `describe-copying' has been modified accordingly.

   Provided by: Romain Francoise <rfrancoise@debian.org>

---
 lisp/help.el |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/lisp/help.el
+++ b/lisp/help.el
@@ -311,7 +311,8 @@
 (defun describe-copying ()
   "Display info on how you may redistribute copies of GNU Emacs."
   (interactive)
-  (view-help-file "COPYING"))
+  (view-file "/usr/share/common-licenses/GPL")
+  (goto-char (point-min)))
 
 (defun describe-gnu-project ()
   "Display info on the GNU project."
